docs/og-analytics/worked-out-examples/worked-out-example-stir-futures-option-normal-moneyness.rst

```diff
@@ -0,0 +1,112 @@
+
+The code to compute the present value simply read as::
+
+    private static final PresentValueNormalSTIRFuturesCalculator PVNFC = PresentValueNormalSTIRFuturesCalculator.getInstance();
+    MultipleCurrencyAmount pv = TRANSACTION_1.accept(PVNFC, NORMAL_MULTICURVES);
+
+The present value calculator is used through the visitor pattern. The result of the present value computation is a **MultipleCurrencyAmount** involving the information about the currency. The present value calculator is further described in :doc:`../PresentValueDiscountingCalculator`.
+
+As a remark the current price of the option is computed as::
+
+    FuturesPriceNormalSTIRFuturesCalculator POC = FuturesPriceNormalSTIRFuturesCalculator.getInstance();
+    double optionPriceQ = TRANSACTION_Q.accept(POC, NORMAL_MULTICURVES) * 100.0;
+
+
+    
+Curve sensitivity
+-----------------
+
+The calculators used for curve sensitivity calculation are::
+
+      PresentValueCurveSensitivityNormalSTIRFuturesCalculator PVCSNFC = PresentValueCurveSensitivityNormalSTIRFuturesCalculator.getInstance();
+      ParameterSensitivityParameterCalculator<NormalSTIRFuturesProviderInterface> PSSFC = new ParameterSensitivityParameterCalculator<>(PVCSNFC);
+      MarketQuoteSensitivityBlockCalculator<NormalSTIRFuturesProviderInterface> MQSBC = new MarketQuoteSensitivityBlockCalculator<>(PSSFC);
+      MultipleCurrencyParameterSensitivity bucketedPv01 =  MQSBC.fromInstrument(TRANSACTION_1, NORMAL_MULTICURVES, BLOCK).multipliedBy(BP1);
+
+The **MQSBC** computes the sensitivity to the market quotes (*bucketed PV01*). The computation is not scaled, i.e. it is for a movement of 1. The last part of the code (**multipliedBy**) multiply it by one basis point to match the market standard. In general, the OG-Analytics library uses absolute numbers (not percent or basis point) everywhere and provide the tools to rescale the output easily.
+
+The format of the output is a *MultipleCurrencyParameterSensitivity* object which can be represented by::
+
+    [EUR-DSCON-OIS, EUR]= (-6.699625499005507E-5, -6.699601771230529E-5, 1.1097176565222359E-10, -3.668758024917107E-9, -0.0025658905357422443, -0.004555359103668548, -0.006086354603993385, 0.17904224681155592, -0.3696466428416493, 0.026504268860874675, -7.840654172613281E-14, 9.633192325368143E-14, 3.080721153798044E-13, 2.4989987634531895E-13, 1.479886424366408E-13, 1.6993761653354066E-13)
+    [EUR-EURIBOR3M-FRAIRS, EUR]= (-0.07843097744832121, 0.0016386932025324356, 7.200414013419107E-4, 0.016153849016546293, 212.57700563629456, -361.1909217716853, -2.2771335549327615E-11, 2.63862064877223E-12, 7.0640044230169935E-12, 1.4665415911199027E-12, 6.039106742059584E-12, 4.3654232078987675E-12, 3.962055990473498E-12, 4.319049695483522E-12, 3.0470051651095408E-12, 2.605778031824316E-12, 1.6592159214426629E-12)
+
```

Comment:
Reformat the output to be more readable, e.g. numbers with 4 decimal places. It can be done easily in Excel.